docs/elasticsearch/tasksapis.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: Tasks API
+parent: Elasticsearch
+nav_order:  8
+has_math: false
+---
+
+# Tasks API
```

Comment:
Hey @keithhc2, the introduction here doesn't need to be substantially longer, but it does need to tell people what this feature is for. What is a task in this context? What are some example tasks that might take a while? Are certain calls to the cluster automatically considered tasks and added to the list, or do I have to add any arguments to my calls?